tests/unit/src/utils/test_rlp.py

```diff
@@ -0,0 +1,113 @@
+import os
+from random import randint
+
+import pytest
+import pytest_asyncio
+from rlp import decode
```

Comment:
plz add this lib to the project requirements (`poetry add {package-name} --group dev`)

Reply:
Included in web3 package

Reply:
yes but what if web3 stops including it in a coming version?

The purpose of the `pyproject.toml` (as `package.json` for npm/yarn for example) is to list everything that is **directly** used by the project.
Then the package manager generates a `.lock` files that actually resolves all packages and their dependencies to find corresponding appropriate versions.

In our case, you will see:

```
$ poetry add rlp --group dev
Using version ^3.0.0 for rlp

Updating dependencies
Resolving dependencies... (0.2s)

Because no versions of rlp match >3.0.0,<4.0.0
 and rlp (3.0.0) depends on eth-utils (>=2.0.0,<3), rlp (>=3.0.0,<4.0.0) requires eth-utils (>=2.0.0,<3).
So, because kakarot depends on both eth-utils (<2.0.0) and rlp (^3.0.0), version solving failed.
```

ie the resolver tells you that not all versions of `rlp` are fine but that we need to fix a limit to the version, so:

```
$ poetry add "rlp<3" --group dev

Updating dependencies
Resolving dependencies... (1.0s)

Writing lock file

No dependencies to install or update
```

and in this later case, it:
- works
- does not actually install anything